tensorflow_cloud/containerize.py

```diff
@@ -56,6 +57,9 @@ def create_docker_file(docker_entry_point,
         destination_dir: Optional working directory in the docker container
             filesystem.
         docker_base_image: Optional base docker image to use. Defaults to None.
+        install_tf_cloud: Boolean flag which indicates whether
```

Comment:
Why special-case this, rather than:

- always installing it, if it is needed most of the time
- letting the user specify it in `requirements_txt`, if it is rarely needed

?

Reply:
> always installing it, if it is needed most of the time
It is not required most of the time, it is needed in the use cases where no `entry_point` is specified. Since this is implementation detail we can change it to always install if we want in the future.

> letting the user specify it in requirements_txt, if it is rarely needed
Since this not something required for the keras model to train, i am not sure users should have to think about this.

Reply:
So it sounds like we want to install it automatically in the cases where no `entry_point` is specified?

In any case, it doesn't seem like an argument we should have in the API. As you said, it is an implementation detail.

Reply:
Yes, `create_docker_file` is not a public API, it is internal and currently consumed from the `run` API.  So, it should be ok to add this parameter?

Reply:
Ok, got it. Makes sense then. Thank you!